code-of-conduct.md

```diff
@@ -1,103 +1,108 @@
-# Creating a Culture of Innovation
+# Creating a culture of innovation
 We aspire to create a culture where people work joyfully, communicate openly
-about things that matter, and provide great services globally. We would like our
+about things that matter, and provide great services globally. We want our
 team and communities (both government and private sector) to reflect on
-diversity of all kinds, not just the classes protected in law. Diversity fosters
+diversity of all kinds, not just the classes protected by the law. Diversity fosters
 innovation. Diverse teams are creative teams. We need a diversity of perspective
 to create solutions for the challenges we face.
 
-This is our code of conduct (adapted from [18F's Code of Conduct](https://github.com/18F/code-of-conduct)).
+This is our code of conduct (adapted from [18F's code of conduct](https://github.com/18F/code-of-conduct)).
 We follow all Equal Employment Opportunity laws and we expect everyone we work
-with to adhere to the [GSA Anti-harrasment Policy](http://www.gsa.gov/portal/directive/d0/content/512516),
-even if they do not work for the Centers for Disease Control and Prevention or
-GSA. We expect every user to follow this code of conduct and the laws and
-policies mentioned above.
+with to adhere to the [US General Services Administation (GSA) anti-harrasment policy](http://www.gsa.gov/portal/directive/d0/content/512516),
```

Comment:
This wasn't part of your changeset, but I think `harassment` is misspelled, here.